.github/pull_request_template.md

```diff
@@ -0,0 +1,58 @@
+```
+INSTRUCTIONS:
+Go through each of these INSTRUCTIONS sections, address them, and then delete them
+```
```

Comment:
We've alreay given instruction for each section. Maybe we can ditch this hint, so one less to delete by builders?